pkg/asset/agent/image/ignition.go

```diff
@@ -492,23 +491,6 @@ func RetrieveRendezvousIP(agentConfig *agent.Config, nmStateConfigs []*v1beta1.N
 	return rendezvousIP, err
 }
 
-func getMirrorFromRelease(releaseImage string, registriesConfig *mirror.RegistriesConf) string {
```

Comment:
this also needs to be renamed in ignition_test.go

Reply:
Fixed.